e2e/cypress/integration/toast/toast_spec.js

```diff
@@ -39,10 +39,8 @@ describe('toasts', () => {
 
     beforeEach(() => {
         // # Click on town-square then off-topic channels in LHS
-        cy.get('#publicChannelList').scrollIntoView().should('be.visible').within(() => {
-            cy.findByText('Town Square').scrollIntoView().should('be.visible').click();
-            cy.findByText('Off-Topic').scrollIntoView().should('be.visible').click().wait(TIMEOUTS.TINY);
-        });
+        cy.get('#sidebarItem_town-square').should('be.visible').click();
+        cy.get('#sidebarItem_off-topic').should('be.visible').click().wait(TIMEOUTS.TINY);
```

Comment:
If not causing the main problem, I suggests not to go back on this type of getting target elements.

Reply:
The problem is the channel keeps juggling between unreads section and public Channel section. Hence made this generic